### ISSUE TITLE
Problem with first colleting metrics

### ISSUE THREAD

**Issue by Esysteme** [PmaControl/PmaControl#55]
`Array and string offset access syntax with curly braces is deprecated`

Happen in Ubuntu 20.04 and PHP 7.4

**Comment by Esysteme** [comment]
/var/www/pmacontrol/App/Controller/Aspirateur.php 658

**Comment by Esysteme** [comment]
```PHP

 if (empty($os)) {
            $version = trim($ssh->exec("cat /etc/debian_version 2> /dev/null"));
            if (!empty($version)) {
                $distributor = trim("Debian");

                switch ($version) {
                    case "4": $codename = "Etch";
                        break;
                    case "5": $codename = "Lenny";
                        break;
                    case "6": $codename = "Squeeze";
                        break;
                    case "7": $codename = "Wheezy";
                        break;
                    case "8": $codename = "Jessie";
                        break;
                    case "9": $codename = "Stretch";
                        break;
                    case "10": $codename = "Buster";
                        break;
                }

                $os = trim("Debian GNU/Linux " . $version . " (" . $codename . ")");
            }
        }
```

**Comment by Esysteme** [comment]
fixed in 0f211cb68e5892df252930c4ce2ef7df7034d192
